spec/features/09_read_reviews_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+feature "User reads reviews" do
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:movie1) { FactoryGirl.create(:movie, title: "Test asdfdasf title") }
+  let!(:movie2) { FactoryGirl.create(:movie, title: "Test fdghdfghgdfh title") }
+  let!(:review1) { FactoryGirl.create(:review, movie_id: movie1.id, user_id: user.id) }
+  let!(:review2) { FactoryGirl.create(:review, movie_id: movie1.id, user_id: user.id) }
+  let!(:review3) { FactoryGirl.create(:review, movie_id: movie2.id, user_id: user.id) }
+  let!(:review4) { FactoryGirl.create(:review, movie_id: movie2.id, user_id: user.id) }
+
+  before do
+    user_sign_in(user)
+  end
+
+  scenario "Do not see reviews on index page" do
+    visit movies_path
+    expect(page).to_not have_content(review1.title)
+    expect(page).to_not have_content(review1.body)
+    expect(page).to_not have_content(review1.rating)
+    expect(page).to_not have_content(review2.title)
+    expect(page).to_not have_content(review2.body)
+    expect(page).to_not have_content(review2.rating)
+  end
+
+  scenario "See a list of reviews on movie 1 show page" do
+    visit movies_path
+    click_on movie1.title
+
+    expect(page).to have_content(review1.title)
+    expect(page).to have_content(review1.body)
+    expect(page).to have_content(review2.title)
+    expect(page).to have_content(review2.body)
+    expect(page).to_not have_content(review3.title)
+    expect(page).to_not have_content(review3.body)
+    expect(page).to_not have_content(review4.title)
+    expect(page).to_not have_content(review4.body)
+  end
+
+  scenario "See a list of reviews on movie 2 show page" do
+    visit movies_path
+    click_on movie2.title
+
+    expect(page).to have_content(review3.title)
+    expect(page).to have_content(review3.body)
+    expect(page).to have_content(review4.title)
+    expect(page).to have_content(review4.body)
+    expect(page).to_not have_content(review1.title)
```

Comment:
Not super necessary to confirm that this data is not on the page. It is not distinguishing this page from another page. It's sort of the equivalent of having `expect(page).to_not have_content("I am a butterfly")` Of course this page doesn't have that, I would not expect it to and that text isn't on the page we are navigating from (confirming we were redirected), so why check for it? Time saving tip :)